test/nn/simplicial/test_sca.py

```diff
@@ -0,0 +1,76 @@
+"""Unit tests for SCA Model."""
+import itertools
+import random
+
+import torch
+from toponetx.classes import SimplicialComplex
+
+from topomodelx.nn.simplicial.sca import AMPSSCA
+
+
+class TestSCA:
+    """Unit tests for the SCA model class."""
+
+    def test_forward(self):
+        """Test the forward method of SCA."""
+        faces = 14
+        node_creation = 17
+        nodes_per_face = 3
+        seed_value = 42
+        random.seed(seed_value)
+        torch.manual_seed(seed_value)
+        # Create a random cell complex of cells with length 3
+        all_combinations = list(
+            itertools.combinations(
+                [x for x in range(1, node_creation + 1)], nodes_per_face
+            )
+        )
+        random.shuffle(all_combinations)
+        selected_combinations = all_combinations[:faces]
+        simplicial_complex = SimplicialComplex()
+        for simplex in selected_combinations:
+            simplicial_complex.add_simplex(simplex)
+
+        x_0 = torch.randn(15, 2)
+        x_1 = torch.randn(35, 2)
+        x_2 = torch.randn(14, 2)
+        down_lap1 = simplicial_complex.down_laplacian_matrix(rank=1)
```

Comment:
Please use conventions for names on laplacians.

`laplacian_down_1`.

Note that you can easily and quickly replace all of these with code editors like visual code.